contrib/vagrant/scripts/07-install-admissionpolicy.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Configures, deletes, and creates kube-dns in the cluster defined in kubectl
+# to configure the spec files. It will use default values from ./helpers.bash
+#######################################
+
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+source "${dir}/helpers.bash"
+
+log "Installing L7 admission policy into kubectl cluster..."
+
+set -e
+
+deployments_dir="${dir}/../deployments"
+vap_file="${deployments_dir}/validatingadmissionpolicy.yaml"
+vapb_file="${deployments_dir}/validatingadmissionpolicybinding.yaml"
```

Comment:
I'm a bit worried about the code duplication here. The two files (the one in `contrib/vagrant` and the one in `install/kubernetes`) will inevitability diverge, unless we have a script that ensures the two are kept in sync.

But I'm wondering, why is this needed? Can we not rely on `helm install` admission policy together with Cilium? Particularly, what will happen with `helm install` if the policy is already present here?

In general, the vagrant scripts are under-maintained and we're not relying on them for much anymore. Therefore I'd be curious for the motivation of adding the validation policy here, especially since the same can be apparently done via `helm install`.

Reply:
> I'm a bit worried about the code duplication here. The two files (the one in `contrib/vagrant` and the one in `install/kubernetes`) will inevitability diverge, unless we have a script that ensures the two are kept in sync.
> 
> But I'm wondering, why is this needed? Can we not rely on `helm install` admission policy together with Cilium? Particularly, what will happen with `helm install` if the policy is already present here?
> 
> In general, the vagrant scripts are under-maintained and we're not relying on them for much anymore. Therefore I'd be curious for the motivation of adding the validation policy here, especially since the same can be apparently done via `helm install`.

Thank you for the code review.
Yes. This is an unnecessary action. I deleted this part of the commit.
Initially, I added these actions to Vagrant because helm didn't start under Vagrant